coalib/misc/Exceptions.py

```diff
@@ -20,10 +15,10 @@ def get_exitcode(exception, log_printer=None):
         exitcode = exception.code
     elif isinstance(exception, VersionConflict):
         log_message = Constants.VERSION_CONFLICT_MESSAGE % str(exception.req)
-        log_printer.log_exception(log_message, exception)
+        LogPrinterMixin().log_exception(log_message, exception)
```

Comment:
There is no benefit in using a class if it only contains a single method. Most probably function calls are slightly faster than static method calls, but I haven't verified that. See https://www.youtube.com/watch?v=o9pEzgHorH0